src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -700,14 +744,17 @@ deleteHistory bh = do
               [SInt (fromIntegral bh)]
 
 initSchema :: BlockHandler SQLiteEnv ()
-initSchema = withSavepoint DbTransaction $ do
-    createBlockHistoryTable
-    createTableCreationTable
-    createTableMutationTable
-    create (domainTableName KeySets)
-    create (domainTableName Modules)
-    create (domainTableName Namespaces)
-    create (domainTableName Pacts)
+initSchema = do
+    vacuumDb
```

Comment:
How expensive is this? In particular for databases that are more than 50% of the file system size and in the order of dozens of GB?

I wonder if this could be responsible for a performance regression that I observed on devnet when testing this PR.

Reply:
I took it out